4_setup_crossval/src/cross_validation_functions.R

```diff
@@ -1,15 +1,10 @@
 
-get_nested_gages<-function(gagesii,nav_distance_km){
-  gage_IDs <- as.character(gagesii$ID)
-  COMIDs <- as.character(gagesii$COMID)
+get_nested_gages<-function(gagesii,nav_distance_km, screened_site_list){
```

Comment:
Can you edit the description using this roxygen format for this function and the following?

  #' @description 
  #'
  #' @param
  #' @return